packages/suite-build/package.json

```diff
@@ -20,11 +20,13 @@
         "build:desktop": "NODE_ENV=production yarn run desktop",
         "lint": "eslint '**/*.{ts,tsx,js}'",
         "type-check": "tsc --build tsconfig.json",
-        "type-check:watch": "yarn type-check -- --watch"
+        "type-check:watch": "yarn type-check -- --watch",
+        "test:unit": "jest --passWithNoTests"
     },
     "dependencies": {
         "@sentry/webpack-plugin": "^1.18.3",
         "@trezor/suite": "1.0.0",
+        "@mobily/ts-belt": "^3.11.0",
```

Comment:
It's used mostly for utility scripts. In case it will used in app code it's very well tree shakable you will only end up with functions you really use in final bundle so it's doesn't matter and it's worth to use even if you end up using only function from whole lib. 

For example, such a simple thing like getting last element from array is still pain in JS. You can simply use `A.last` without any worries that in will add to your bundle lot of uncessary code. And thats only one of many examples what could be super useful for you.